# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ntfs2btrfs/memory-include.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ntfs2btrfs-20240115/src/ntfs.h.vanilla	2024-12-06 16:05:59.573333349 +0100
+++ ntfs2btrfs-20240115/src/ntfs.h	2024-12-06 16:06:06.646666686 +0100
@@ -23,6 +23,7 @@
 #include <string>
 #include <list>
 #include <functional>
+#include <memory>
 
 #ifdef _WIN32
 #include <windows.h>
